packages/beeq/src/components/alert/bq-alert.tsx

```diff
@@ -125,31 +133,44 @@ export class BqAlert {
   /** Method to be called to hide the alert component */
   @Method()
   async hide(): Promise<void> {
-    this.handleHide();
+    await this.handleHide();
   }
   /** Method to be called to show the alert component */
   @Method()
   async show(): Promise<void> {
-    this.handleShow();
+    await this.handleShow();
   }
 
   // Local methods
   // Internal business logic.
   // These methods cannot be called from the host element.
   // =======================================================
 
-  private handleHide = () => {
+  private handleHide = async () => {
     const ev = this.bqHide.emit(this.el);
     if (!ev.defaultPrevented) {
+      await leave(this.alertElement);
       this.open = false;
+      this.handleTransitionEnd();
     }
   };
 
-  private handleShow = () => {
+  private handleShow = async () => {
     const ev = this.bqShow.emit(this.el);
     if (!ev.defaultPrevented) {
       this.open = true;
+      await enter(this.alertElement);
+      this.handleTransitionEnd();
+    }
+  };
```

Comment:
Same for here, we remove the `.is-hidden` class before the enter animation

```suggestion
  private handleShow = async () => {
    const ev = this.bqShow.emit(this.el);
    if (!ev.defaultPrevented) {
      this.open = true;
      this.el.classList.remove('is-hidden');
      await enter(this.alertElement);
      this.handleTransitionEnd();
    }
  };
```

Reply:
and with this, we can remove from the `<Host>` the conditional to apply the `.is-hidden` class:

![CleanShot 2024-01-12 at 19 05 04@2x](https://github.com/Endava/BEEQ/assets/328492/cd1d4bbd-5f6f-405f-8bd7-70c73c7404fa)